edb/schema/pseudo.py

```diff
@@ -70,7 +70,7 @@ def __eq__(self, other: s_types.Type) -> bool:
                 self.name == other.name)
 
 
-AnyT = typing.TypeVar('AnyT', bound='Any')
+AnyT = typing.TypeVar('AnyT', 'Any', 'AnyTuple')
```

Comment:
@1st1, if you changed this instead of doing the other changes to `pseudo.py`, that fixes the problem that Mypy was finding.

Reply:
Lukasz, I've already merged https://github.com/edgedb/edgedb/commit/f30744d3ef4bd9f893ab56a5b5d1b705be215077. While it's a bit more impactful I also typed the `pseudo.py` while at it, so let's just close this one.

Reply:
Good to know this wasn't a bug in mypy, btw. I thought it was.